backend/models/employeeModel.js

```diff
@@ -27,9 +27,14 @@ const employeeSchema = new Schema({
     type: String,
   },
   numer_konta: {
-    type: Number,
+    type: String,     //bug fix - bank account can possibly start with zero
```

Comment:
Please in the future indicate the change with new jira id. It is hard to check when it was changed.